publisher/monitor/price/monitor-price.py

```diff
@@ -15,7 +15,7 @@
 
 
 PRICE_TOLERANCE = 0.1  # in percent
-TIME_TOLERANCE = 600  # in seconds
+TIME_TOLERANCE = 1200  # in seconds
```

Comment:
What is reasoning for increasing time tolerance to 20 minutes. That is 4 missed retries/publishing. Open-minded (maybe Starkware downtimes; funny enough a test failed that I reran and passed I think bc of the Starkex latency problem yesterday) but just looking for more clarity on reasoning.

Reply:
Yes, ideally it should be 5 minutes actually because publishers publish at offset times so overall the most recent time updated should never be older than 5 minutes. In practice testnet transactions sometimes take 10 minutes to be processed, and I don't want to get used to alarm bells ringing as that will desensitize us for when something actually goes wrong.

Reply:
That makes sense. I had thought 15 would be more appropriate for aa 5 minute interval but in the context of testnet getting as high as 10 for confirmation, this makes sense. Approving now